container-storage-setup.sh

```diff
@@ -49,6 +49,7 @@ _STATUSFILE_NAME="status"
 # Read man page for a description of currently supported options:
 # 'man container-storage-setup'
 
+_DEFAULT_CONTAINER_THINPOOL=docker-pool
```

Comment:
Not sure why do we need it. Is it just a cleanup/fix for some other issue.

Reply:
See the use of it in `select_auto_storage_driver`